### PR TITLE
Update notebooks for lilio, add notebooks to docs

### LINKED ISSUE #23: Add example notebooks to documentation
We have some really nice example notebooks. It would be nice if these are part of the documentation. We could e.g. use [nbsphinx](https://nbsphinx.readthedocs.io/en/0.8.12/) and/or [myst-nb](https://github.com/AI4S2S/lilio/blob/main/notebooks/all_about_the_calendar.ipynb)

### PR DESCRIPTION
**Pull request by BSchilperoort**
In this PR:
- [x] Add action to ensure notebooks are clean
- [x] Add notebooks to documentation, execute them & ensure no errors are raised
- [x] Update notebooks to Lilio
- [x] Make new notebook about calendar shorthands 

Closes #23

### CODE REVIEW

**Comment by BSchilperoort** [comment]
image
Action works as intended :)

File: docs/notebooks/all_about_the_calendar.ipynb [hunk 1]
```diff
@@ -0,0 +1,508 @@
+{
```
Comment (geek-yang) [inline docs/notebooks/all_about_the_calendar.ipynb:1]: <p>Need to fix the display of figure (extension in file name is case sensitive).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/lilio/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (geek-yang) [inline docs/notebooks/all_about_the_calendar.ipynb:1]: <p>This is really awesome!!! Love the idea!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/lilio/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

**Review by geek-yang — APPROVED** [review]
Awesome work! Many thanks for the renovation of notebooks. The use of emoji really wins my heart and that definitely makes me feel more attracted by the markdown cells of notebooks, rather than code blocks only.

Only one small comment regarding the broken figure, which can be easily fixed in a second. Thanks again for your effort!

**Comment by BSchilperoort** [comment]
Thanks for the review, Yang!

**Pull request merged**